### PR TITLE
#138 Csenge new comments

### PR DESCRIPTION
**Pull request by baxgas**
Just some small fixes after Csenge's comments on new version

### CODE REVIEW

File: app/components/RadioPlayer.vue
Comment (gammaw) [inline app/components/RadioPlayer.vue:116]: Do we disable the sand clock completely?

**Review by gammaw — COMMENTED** [review]
@baxgas see inline comment about the sand clock.

File: app/components/RadioPlayer.vue
Comment (baxgas) [reply app/components/RadioPlayer.vue:116]: Csenge told me also she finds it unneeded and distracting

Comment (baxgas) [reply app/components/RadioPlayer.vue:116]: Otherwise it is of course open to discussion on the next meeting

Comment (gammaw) [reply app/components/RadioPlayer.vue:116]: Oh, that's a bold change :) I would say, let's wait until the meeting before we merge this change. And if we decide to get rid of the sand clock, I think it's better that we delete it from code (we have it in the history in case we want to bring it back some time in the future).

**Review by pvj — COMMENTED** [review]
Let's keep the hourglass!!!!

For everything else I'm in favor of merging this.

**Review by pvj — APPROVED** [review]

**Review by gammaw — APPROVED** [review]
Save hourglass succeeded! :)

**Pull request merged**
